### ISSUE TITLE
SVG file export without Ko info

### ISSUE THREAD

**Issue by simonguoxm** [bernds/q5Go#77]
When I export the SVG file, Q5go can only export the current board view. There is no info about all steps. Usually Ko info is added outside of the board, such as 75 at 32, etc.

**Comment by bernds** [comment]
Yes, the problem is I couldn't quite figure out how to format this nicely in SVG. So it's left open for now.
